source/_integrations/ecobee.markdown

```diff
@@ -190,6 +193,15 @@ Enable/disable automatic daylight savings time.
 | `entity_id`            | yes      | ecobee thermostat on which to set daylight savings time mode |
 | `dst_enabled`          | no       | true or false                                     |
 
+### Service `ecobee.set_humidifier_mode`
+
+Set the humidifier mode.
+
+| Service data attribute | Optional | Description                                       |
+| ---------------------- | -------- | ------------------------------------------------- |
+| `entity_id`            | no       | ecobee thermostat on which to set the humidifier mode |
```

Comment:
Entity ID is optional, as this service is registered with `async_register_entity_service`.

```suggestion
| `entity_id`            | yes      | ecobee thermostat on which to set the humidifier mode |
```